docs/main/home/introduction.mdx

```diff
@@ -78,11 +78,11 @@ Keep reading to know more about how Dyte works, or [**click here to jump to our
 
 <!-- ![Diagram explaining how Dyte works](./how-dyte-works.png) -->
 
-Dyte is based largely on the WebRTC protocol, and the features and optimizations we build on top of it. Although the protocol is supposed to be peer-to-peer, anyone who has tried implementing it by themselves would know that the implementions get real messy, real fast. To have a real working solution, servers with SFU, STUN, TURN, etc capabilities are required to optimize for user bandwidth and performance, route the data packets correctly and maintain state of certain client side configurations and preferences.
+Dyte is based largely on the WebRTC protocol, and the features and optimizations we build on top of it. Although the protocol is supposed to be peer-to-peer, anyone who has tried implementing it by themselves would know that the implementations get real messy, real fast. To have a real working solution, servers with SFU, STUN, TURN, etc capabilities are required to optimize user bandwidth and performance, route the data packets correctly and maintain a state of specific client-side configurations and preferences.
 
-On top of managing this basic communication using media streams, Dyte offers additional features on top such as chat, polls, plugins, recording and streaming, and many others.
+On top of managing this basic communication using media streams, Dyte offers additional features such as chat, polls, plugins, recording and streaming, and many others.
 
-Dyte's mission is to make this technology available to developers for easy use, and towards that we provide several implementations of Dyte's real-time communications services, which enable developers to integrate a meeting within a few lines of code without having any previous knowledge about media, streaming, or RTC, but also allow them to go deeper if they so wish. The features offered on top of the basic communication service are also in a developer's control - which ones do they want to pick and to what level do they want to customize it.
+Dyte's mission is to make this technology available to developers for easy use. Towards that, we provide several implementations of Dyte's real-time communications services, which enable you to integrate a meeting within a few lines of code without having any previous knowledge about media, streaming, or RTC but also allow you to go deeper if you so wish. The features offered above the basic communication service are also under the developer's control. You can pick which features you want and to what level you want to customize them.
```

Comment:
Dyte's mission is to make this technology available to developers for easy use. We provide several implementations of Dyte's real-time communications services, which enable you to integrate a meeting within a few lines of code - without any prerequisite knowledge about media, streaming, or RTC. Nonetheless, we allow you to dive deeper into the intricacies if you so wish. You can control the features offered above; you can pick which features you want, and customize them to your liking.